lib/dialects/abstract/connection-manager.js

```diff
@@ -6,17 +6,24 @@ const _ = require('lodash');
 const Utils = require('../../utils');
 const debug = Utils.getLogger().debugContext('pool');
 const semver = require('semver');
-const timers = require('timers');
 
 const defaultPoolingConfig = {
   max: 5,
   min: 0,
   idle: 10000,
   acquire: 10000,
-  evict: 60000,
+  evict: 10000,
```

Comment:
We need to set `evict ~= idle` , reason being even if idle time is crossed its the evictor that will remove those idle connections.

More details https://github.com/coopernurse/node-pool/issues/178

Reply:
Maybe a good idea to mention this in the API docs. (perhaps just a link to that issue)

Reply:
@sushantdhiman by saying `evict ~= idle`, do you mean `evict ≈ idle` ? i think it's fine to set evict to some value a little smaller than idle.

Reply:
@xianxingg as mentioned in https://github.com/coopernurse/node-pool/issues/178#issuecomment-327110870

They should be close (or even evict < idle) because only evictor remove idle connection from pool